index.html

```diff
@@ -10,22 +10,23 @@
         <title>Friday</title>
     </head>
     <body>
-        <header>
-            <div class="header-container">
+        <header id="header-bg">
+            <div class="header-container" id="header">
                 <nav class="navigation">
                     <ul>
-                        <li><a><img src="images/image_logo.png">Friday</a></li>
-                        <li><a>About</a></li>
-                        <li><a>Contact</a></li>
-                        <li><a><img src="#"></a></li>
+                        <!-- Need help with alignment, please. -->
+                        <li><a><img src="images/image_logo.png" alt="Friday Logo" id="logo"></a></li>
+                        <li class="fs-4"><a>About</a></li>
+                        <li class="fs-4"><a>Contact</a></li>
+                        <li><a><img src="images/instagram_logo.png" alt="Contact Icon" id="contact"></a></li>
                     </ul>
                 </nav>
             </div>
         </header>
         <main> 
             <section class="banner">
                 <h2>Friday, plan your next week</h2>
-                <div class="background image"><img src="images/image_banner.jpg" alt="Lady smiling happily on street"></div>
+
```

Comment:
hi Daniel, it sounds that you overwrote this change for the banner image from Sam. Please get pull the latest commits from main repository and merge yours